validation/bickley_jet/disk_time_series.jl

```diff
@@ -0,0 +1,63 @@
+module DiskTimeSerieses
+
+using JLD2
+
+using Oceananigans
+using Oceananigans.Grids: AbstractGrid
+using Oceananigans.Fields: offset_data
+
+import Oceananigans.Grids: nodes
+
+struct DiskTimeSeries{F, N, G, L, I, T}
```

Comment:
I'm a bit puzzled as to what this is doing, but I am probably behind  on emails. Sorry.

Reply:
This is a rogue abstractions that will hopefully eventually find its way into the source. It provides an abstraction for a time series of data "on disk" that's helpful for post-processing.

Reply:
I experience human validation for including this code because @ali-ramadhan has included python code in the rigid lid validation experiment *gasp*

Reply:
Thanks and looking forward to seeing how this evolves.